salt/orch/update-etc-hosts.sls

```diff
@@ -0,0 +1,23 @@
+update_grains:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: saltutil.refresh_grains
+
+update_mine:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: mine.update
+    - require:
+      - salt: update_grains
+
+etc_hosts_setup:
+  salt.state:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - concurrent: True
```

Comment:
You are right. I thought it was for running that state concurrently in several minions, but it is on the same minion. Removing this line...